fourmolu.yaml

```diff
@@ -0,0 +1,18 @@
+indentation: 2
+column-limit: 90
```

Comment:
I don't want to fight defaults too much, so I'd like to use this config:

```yaml
indentation: 2
column-limit: 90
```

Motivation for changes:

 * indentation: honestly I think people will change the locks on the office if I don't
 * column-limit: I'm a bit conflicted on this one. I'd like us to try to hit 80/90. Though sometimes it really doesn't make sense to stay within. We'll have to see how this plays out.